.ci/ci-fast-return.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2017-2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Perform CI 'fast path return' checks.
+# Returns: 1 for 'fast return OK', 0 for 'do not fastpath'.
+
+set -e
+
+[ -n "$DEBUG" ] && set -x
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+# If no branch specified, compare against the master.
+# The 'branch' var is required by the get_pr() lib functions.
+branch=${branch:-master}
+
+# Check if any files changed by the PR are not in the 'fast path'
+# CI list.
+# Return:
+#  0 - all files are fastpath
+#  1 - at least one non-fastpath file found
+any_nonfast_files()
+{
+	# The branch is the baseline - ignore it.
+	[ "$specific_branch" = "true" ] && return 1
+
+	files=$(get_pr_changed_file_details || true)
+
+	# Strip off the leading status - just grab last column.
+	files=$(echo "$files"|awk '{print $NF}')
+
+	# no files were changed - I guess we can skip the CI then?
+	[ -z "$files" ] && info "No files found" && echo "1" && return
+
+	# which egrep filename patterns do *NOT* need to be CI checked.
+	local pats=""
+	pats+="-e ^CODEOWNERS$ "
+	pats+="-e ^LICENSE$ "
+	pats+="-e *.\.md "
```

Comment:
As mentioned on https://github.com/kata-containers/ci/issues/192, normally docs can be ignored. But not if they are https://github.com/kata-containers/documentation/tree/master/install as those *must* be CI checked. Could you add a rule for those here?